liquidctl/driver/msi_7zw0led.py

```diff
@@ -0,0 +1,43 @@
+"""liquidctl drivers for MSI liquid coolers.
+
+Supported devices:
+
+- MPG CORELIQUID K360
+
+Copyright (C) 2020–2021  Andrew Udvare and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+from liquidctl.driver.usb import UsbHidDriver
+
+_REPORT_LENGTH = 64
+
+
+class MSI7ZW0LED(UsbHidDriver):
+    """7ZW0LED is the internal name in the MSI Center app."""
+
+    SUPPORTED_DEVICES = [
+        (0x0db0, 0xb130, None, 'MSI MPG Series CORELIQUID K360', {}),
+    ]
+
+    def get_status(self, **kwargs):
+        buf = bytearray(_REPORT_LENGTH)
+        buf[1] = 0xd0
+        buf[2] = 0x31
+        self.device.clear_enqueued_reports()
+        self.device.write(buf)
+        array = bytes(self.device.read(_REPORT_LENGTH))
+        if array[1] != 0x31:
+            raise ValueError('Unexpected value in response buffer')
```

Comment:
Assuming this is a rare event, maybe an assertion would be more appropriate?

(If on the other hand this is a common occurrence, then we need to handle it some other way, perhaps with retries).